django/contrib/postgres/fields/array.py

```diff
@@ -179,6 +181,19 @@ def as_sql(self, qn, connection):
 
 
 @ArrayField.register_lookup
+class ArrayIn(In):
```

Comment:
This specific type didn't like having it's array elements unwrapped and processed individually. So I reverted the change I made to the base `In` type by reimplementing the QueryWrapper behaviour here. So, no new tests.